### ISSUE TITLE
proot error: '/system/bin/toolbox' is a 64-bit program whereas this version of proot handles 32-bit programs only

### ISSUE THREAD

**Issue by Gurren-Joestar** [termux/proot#197]
**Problem description**

Toolbox cant be executed due to proot being 32 bits, similar issues with other custom distros 
(i.e. https://github.com/Hax4us/TermuxAlpine 
 https://github.com/TermuxArch/TermuxArch)
![image](https://user-images.githubusercontent.com/71229258/138110622-3d8504bc-79d8-4697-9f7a-17be9b1deb8f.png)

CPU and ABI seems to be 64 bits (aarch64 - arm64-v8a), I only suspect the OS version could be 32 bits, but Im not sure how to know that, if my OS is 32 bits I would like to know how to get a proot x64 so toolbox works and proot custom distros works so.

**Steps to reproduce**

install proot
execute toolbox inside proot

**Expected behavior**

toolbox works

**Additional information**

```
$ uname -a
Linux localhost 4.14.113-22145712 #1 SMP PREEMPT Sat Jul 31 05:45:23 KST 2021 armv8l Android

$ getprop | grep cpu
[ro.odm.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist64]: [arm64-v8a]
[ro.product.cpu.abi]: [arm64-v8a]
[ro.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.product.cpu.abilist64]: [arm64-v8a]
[ro.vendor.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist64]: [arm64-v8a]

$ file /system/lib64/
/system/lib64/: directory
```

more information:

<<< System >>>

Model: SM-A515F
Board: exynos9611

<<< CPU >>>

Scaling Governor: schedutil
Supported ABIs: arm64-v8a, armeabi-v7a, armeabi
Supported 32-bit ABIs: armeabi-v7a, armeabi
Supported 64-bit ABIs: arm64-v8a

<<< Android >>>

Android Version: 11
API Level: 30
Kernel Architecture: aarch64
Kernel Version: 4.14.113-22145712

Aditionally I would like to know why `uname -m` output `armv8l` instead of aarch64, using other terminals apps like pydroid give me this result:

```
/storage/emulated/0 $ uname -m
aarch64
```

Alternative Using TermOne Plus -terminal emulator (https://gitlab.com/termapps/termoneplus)

```
:/ $ uname -m
aarch64
```

**Comment by xeffyr** [comment]
@Yonle No, this is normal 64 bit Android OS. Please read the issue carefully, especially stats:
```
[ro.product.cpu.abi]: [arm64-v8a]
[ro.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.product.cpu.abilist64]: [arm64-v8a]
```

Btw, nothing to do with Termux either. That's entirely setup issue.

**Comment by xeffyr** [comment]
> I studied the output and still cannot phantom how you knew this is a NOT a mixed architecture issue.

Do not read **between** lines. In my [first comment](https://github.com/termux/proot/issues/197#issuecomment-947812813) there is no statement that this is *not mixed architecture issue*.

This is a standard 64 bit installation of Android OS (see ABIs in stats), but Termux environment is somehow 32bit. The only ways how this is possible are:
* Wrong Termux app build variant. For example, if using GitHub Actions builds, it is possible to accidentally pick armeabi-v7a build instead of armv8-a or universal. Even though we provide 2 builds (the mentioned GitHub Actions and F-Droid), the official supported version is the F-Droid one only (https://f-droid.org/repo/com.termux_117.apk).
* Custom bootstrap installation - it is unknown what has been done before issue happened.
* Very unlikely but still possible failure of architecture detection in Termux app.
